addons/aircraft/functions/fnc_gau8_muzzleEffect.sqf

```diff
@@ -13,18 +13,19 @@
  * Return value:
  * None
  *
+ * Example:
+ * QUOTE(_this call FUNC(gau8_muzzleEffect));
```

Comment:
Should write it out full, eg. `[unit, "weapon", "muzzle", "ammo", "magazine", projectile] call ace_aircraft_fnc_gau8_muzzleEffect;`